README.md

```diff
@@ -122,4 +139,23 @@ Pull requests are very welcome! Make sure your patches are well tested. Ideally
 
 5. Create new Pull Request
 
-6. Add the test results in the PR
\ No newline at end of file
+6. Add the test results in the PR
+
+# Supported features
+
+1. Dynamic PVC/PV creation and deletion with ReadWriteMany capability
+2. Deployment creation and deletion which will mount/unmount the file storage volumes.
+3. ReplicationController creation and deletion which will mount/unmount the file storage volumes.
+4. StatefulSet creation and deletion which will mount/unmount the file storage volumes.
```

Comment:
Just keep PV mount and mount
PV creation and deletion with RWM access. 
Mount/Unmount of file shares to or from the VPC target.